### DISCUSSION TITLE
I'd like to have ...

### DISCUSSION THREAD

**Discussion by Harry1002** [mocaccinoOS/mocaccino#127] [Ideas]
Hello,

Sabayon was a distribution without frills and so is Mocaccino. I used Sabayon from DVD and use Mocaccino from USB stick as desktop live system. But there was more software included to Sabayon. I'd like to have LibreOffice, a second browser (Chromium?), GIMP and a better PDF-viewer.

Harry

**Comment by joostruis** [comment]
Hi there. It is all there already you just need to install it.
apps/chromium can be installed from the desktop repository that is already enabled. Additional software can be installed after you enable the community-repository.

```bash
sudo luet install repository/mocaccino-community-stable
```
After that you can find libreoffice:

```bash
sudo luet search libreoffice
```
and install it:
```bash
sudo luet install apps/libreoffice
```

Enjoy!

**Comment by Harry1002** [comment]
Hi,

I use Mocaccino from DVD (or USB stick) as **live system** on a machine with 8 GB of RAM. I cannot install software, because my machine runs out of space. So I'd like to have some widespread applications like libreoffice on the ISO image, ready to go.

I tried
`sudo luet install htop`
because htop is a tiny piece of software. Luet checked for available upgrades, found 37 new versions and started to download. My /var/tmp/luet directory ran out of space at 4/37 and so I stopped.

This is why I'd like to have more software included on the ISO image.

Harry

**Comment by joostruis** [comment]
I'm sorry but we did not design this for running in live session like you require.
